bftclient/include/bftclient/fake_comm.h

```diff
@@ -133,8 +133,20 @@ class FakeCommunication : public bft::communication::ICommunication {
   BehaviorThreadRunner runner_;
   std::thread fakeCommThread_;
 };
+std::vector<uint8_t> replayConfig(const MsgFromClient& msg);
```

Comment:
I think a better name for this would be `createReply`.

Reply:
changed it. Thank you.